ui/src/App.tsx

```diff
@@ -6,32 +6,33 @@ import "react-big-calendar/lib/css/react-big-calendar.css";
 import { CalendarEvent } from "./configs/CalendarEvent";
 import s from "./App.module.css";
 
-const currentHour: number = new Date().getHours();
-const currentMinute: number = new Date().getMinutes();
-const padNumberWith0: Function = (num: Number): string =>
-  num.toString().padStart(2, "0");
-const defaultStartTime: string = `${padNumberWith0(
-  currentHour,
-)}:${padNumberWith0(currentMinute)}`;
-const defaultEndTime: string = `${padNumberWith0(
-  currentHour + 1,
-)}:${padNumberWith0(currentMinute)}`;
+const App = () => {
+  const currentHour: number = new Date().getHours();
+  const currentMinute: number = new Date().getMinutes();
+  const padNumberWith0: Function = (num: Number): string =>
```

Comment:
We typically don't use numbers in function names. Let's rename this to be `padNumberWithZero`

Reply:
Once this is addressed this PR is good to go!

Reply:
I think we missed this comment ^